python/lsst/ip/isr/brighterFatterKernel.py

```diff
@@ -504,6 +524,27 @@ def toTable(self):
 
         return tableList
 
+    def repackCorrelations(self, amp, correlationShape):
```

Comment:
It's nice to have a helper function for repacking the correlations. My general worry is, wouldn't we need to repack also other quantities like the fluxes (means) at the same time? Are we running out of sync somehow between the different arrays? The PTC code has an associated "covariances weights" array, although it seems that it is not used in the BFK code.

Reply:
For the means and variances, the BF code uses the raw versions.  Previously, the covariances had the same length as those, which is likely why I used the raw values.  I've filed a ticket, DM-36344, to put more thought into this choice; If the final values have the same length as the covariances now, those may be the more natural choice.
What are the weights used for in the PTC?  I'd have to think more about the BF algorithm to determine if we should be using those as well.

Reply:
The "covariances weights" in the PTC code are calculated from Pierre's code (using the variance of the variance formula ``2*Var/Npix``), and then used in the fit to the ``FULLCOVARIANCE`` model.